### ISSUE TITLE
cz-cli may be not necessary

### ISSUE THREAD

**Issue by vincent-ng** [callumacrae/find-node-modules#14]
cz-cli takes many disk size which can be removed I think.

```bash
$:/node_modules/find-node-modules$ ls
README.md  cz-cli  index.js  package.json  test  test.js

$:/node_modules/find-node-modules$ du -h --max-depth=1 .
116M    ./cz-cli
0       ./test
```

**Comment by callumacrae** [comment]
That directory isn't coming from this project: https://github.com/callumacrae/find-node-modules/blob/master/package.json

Are you using commitizen?

**Comment by vincent-ng** [comment]
Yes, I know it's not declared in the package.json.
But I can find the "cz-cli" in the node_modules.

```bash
$ npm i find-node-modules
$ ls node_modules/find-node-modules
README.md  cz-cli  index.js  package.json  test  test.js
```

I suppose it is accidentally published to the npm.

**Comment by callumacrae** [comment]
Oh I see, it was in the npm version! I'm not sure how it got there - maybe I was testing something and forgot to delete it 😬 

I've deleted it and published as 2.1.2.
